docs/build-install/cpe.md

```diff
@@ -1,97 +1,112 @@
 
 ## CPE in a container
-!!! info
-    Currently it is mandatory to export the EDF path manually
-    ```
+
+[](){#ref-cpe-versions}
+### Available versions
+
+The `PrgEnv-gnu` and `PrgEnv-cray` programming environments are provided as separate containers, instead of having both in one container, named `gnu-$version` and `cray-$version` respectively.
+The `version` is the CPE version in the container.
+
+|                 | `zen2`   | `gh200` |
+|-----------------|----------|---------|
+| `cpe-gnu-24.7`  | ❌       | ✅      |
+| `cpe-cray-24.7` | ❌       | ✅      |
+
+!!! warning "only available on gh200"
+    The CPE container is only provided on systems with the "[container engine][ref-container-engine]" container runtime, which is currently the Grace-Hopper systems [Daint][ref-cluster-daint], [Clariden][ref-cluster-clariden] and [Santis][ref-cluster-santis].
+
+### How to use
+
+!!! info "Before you start"
+    To use the CPE containers as documented below, you need to set the `EDF_PATH` environment variable.
+    ```console
     export EDF_PATH=/capstor/scratch/cscs/anfink/shared/cpe/edf:$EDF_PATH
     ```
+    Note that the current location of the EDF files is temporary, to work around a file system bug.
+    The environment variable __will not be required__ once this issue is fixed.
 
-To start a container with the cray programming environment, there are pre-defined EDF toml files. Currently they are stored in `/capstor/scratch/cscs/anfink/shared/cpe/edf`, but this will change in the future to a different location.
-Every toml file in this edf path is a programming environment that can be started
-```
-[daint][anfink@daint-ln001 ~]$ ls -lh /capstor/scratch/cscs/anfink/shared/cpe/edf
-total 8.0K
--rw-r--r--+ 1 anfink csstaff 175 Apr 23 11:48 cpe-cray-24.07.toml
--rw-r--r--+ 1 anfink csstaff 174 Apr 23 11:32 cpe-gnu-24.07.toml
-```
-Following the naming scheme from the output, you can spawn a container with
-```
-srun --environment=cpe-cray-24.07 --pty bash
+To start a session with the CPE (see [available-versions][ref-cpe-versions] above):
+```console
+$ srun --environment=cpe-cray-24.07 --pty bash
 ```
 Once the container starts up you can directly use the programming environment, because there will be modules loaded by default at startup.
-```
-[daint][anfink@daint-ln001 ~]$ srun -p debug --environment=cpe-gnu-24.07 --pty bash
-[daint][anfink@nid005417 /]$ module list
 
-Currently Loaded Modules:
-  1) craype-arm-grace   2) craype-network-ofi   3) xpmem/2.9.6   4) gcc-native/13.2   5) craype/2.7.32   6) PrgEnv-gnu/8.5.0   7) cray-mpich/8.1.30   8) cuda/12.6   9) craype-accel-nvidia90
+```console
+$ srun -p debug --environment=cpe-gnu-24.07 --pty bash
+$ module list
 
+Currently Loaded Modules:
+  1) craype-arm-grace     4) cce/18.0.0          7) cray-mpich/8.1.30
+  2) craype-network-ofi   5) craype/2.7.32       8) cuda/12.6
+  3) xpmem/2.9.6          6) PrgEnv-cray/8.5.0   9) craype-accel-nvidia90
```

Comment:
This is the output of cpe-cray-24.07, not as susggested in the command above cpe-gnu-24.07